docs/technical_reference_guide/tables/options/Geometry Attic Type.tex

```diff
@@ -1,8 +1,14 @@
-\begin{customLongTable}{ |p{3cm}|p{2.75cm}|p{2.75cm}| }
+\begin{customLongTable}{ |p{5cm}|p{3cm}|p{3cm}|p{3cm}| }
 {Geometry Attic Type options and properties that vary for each option} {table:hc_opt_geometry_attic_type}
```

Comment:
"options and properties that vary for each option" -- I think it should simply say "options and properties" now. See `geometry_roof_pitch` below for an example of a property that is displayed that doesn't vary for each option.

<img width="555" height="233" alt="Image" src="https://github.com/user-attachments/assets/a47422a1-80a1-4baa-b61a-ab1557ebe165" />

Reply:
https://github.com/NREL/resstock/pull/1514/commits/a53fe8eca7f0ad9ba8bab9472c7056725d68631f